src/__screenshot_tests__/private-text-preset-vars-screenshot-test.tsx

```diff
@@ -0,0 +1,25 @@
+import {openStoryPage, screen} from '../test-utils';
+
+const SKINS = ['Movistar', 'Vivo-new', 'O2-new'] as const;
```

Comment:
Tested only in these skins because I don't see any value in adding all the possible ones here